website/content/en/v0.7.0/getting-started/getting-started-with-eksctl/cloudformation.yaml

```diff
@@ -57,4 +53,20 @@ Resources:
               - ec2:DescribeInstanceTypes
               - ec2:DescribeInstanceTypeOfferings
               - ec2:DescribeAvailabilityZones
+          - Effect: Allow
+            Resource: "*"
+            Action:
+              - ec2:RunInstances
+              - ec2:TerminateInstances
+              - ec2:DeleteLaunchTemplate
+            Condition:
+              StringEquals:
+                "ec2:ResourceTag/karpenter.sh/discovery": !Ref ClusterName
+          - Effect: Allow
+            Resource: !Ref KarpenterNodeRole
+            Action:
+              - iam:PassRole
+          - Effect: Allow
+            Resource: !Sub "arn:${AWS::Partition}:ssm:*:*:parameter/aws/service/*"
```

Comment:
When calling `aws ssm get-parameters-by-path --path /aws/service/list`, the parameters don't include an account ID.  Should this resource be updated to:

```
!Sub "arn:${AWS::Partition}:ssm:*::parameter/aws/service/*"
```